eng/intellisense.targets

```diff
@@ -1,5 +1,9 @@
 <Project>
 
+  <PropertyGroup>
+    <UseCompilerGeneratedDocXmlFile Condition="'$(UseCompilerGeneratedDocXmlFile)' == ''">true</UseCompilerGeneratedDocXmlFile>
+  </PropertyGroup>
+
   <PropertyGroup Condition="'$(UseCompilerGeneratedDocXmlFile)' != 'true'">
```

Comment:
We should add an error below in `ChangeDocumentationFileForPackaging` for when `UseCompilerGeneratedDocXmlFile` is not `true` and `IntellisensePackageXmlFilePath` is not set (the file doesn't exist).  That will prevent the case where folks think they're getting one thing but aren't.

Reply:
In other words, remove `'$(IntellisensePackageXmlFilePath)' != ''` from the target condition, add it to the ItemGroup, then add a warning that's the inverse:

```
<Warning Condition="'$(IntellisensePackageXmlFilePath)' == ''" Text="`UseCompilerGeneratedDocXmlFile` was set to `$(UseCompilerGeneratedDocXmlFile)`, but the doc-team provided file was not found" />
```

Reply:
Done. I added the warning to a separate target though.